src/tests/class_tests/openms_gui/CMakeLists.txt

```diff
@@ -100,9 +100,9 @@ if(HAS_XSERVER)
   # Add GUI tests
   foreach(_gui_test ${GUI_executables_list})
     # moc the test file
-    QT5_WRAP_CPP(_mocced_test_file source/GUI/${_gui_test}.cpp OPTIONS ${BOOST_MOC_ARGS})
+    #QT5_WRAP_CPP(_mocced_test_file source/GUI/${_gui_test}.cpp OPTIONS ${BOOST_MOC_ARGS})
     # define source file dependency
-    set_source_files_properties(source/GUI/${_gui_test}.cpp PROPERTIES OBJECT_DEPENDS ${_mocced_test_file})
+    #set_source_files_properties(source/GUI/${_gui_test}.cpp PROPERTIES OBJECT_DEPENDS ${_mocced_test_file})
```

Comment:
maybe we would have needed this for the TOPPView_test ? seems that it needs to be mocced as well?